include/oneapi/dpl/experimental/kt/two_pass_scan.h

```diff
@@ -56,45 +57,108 @@ sub_group_masked_scan(const SubGroup& sub_group, MaskOp mask_fn, InitBroadcastId
             value = binary_op(partial_carry_in, value);
         }
     }
-    value = binary_op(init, value);
-    // Adjust for an exclusive scan if requested
-    if constexpr (!Inclusive)
+    if constexpr (InitPresent)
```

Comment:
Can this conditional be combined with the one below?

Reply:
Yes, thanks